actions/class.PropertiesAuthoring.php

```diff
@@ -106,7 +108,19 @@ public function addClassProperty(): void
             'disableIndexChanges' => $this->isElasticSearchEnabled()
         ];
 
-        $propFormContainer = new tao_actions_form_SimpleProperty($class, $class->createProperty('Property_' . $index), $options);
+        $newProperty = $class->createProperty('Property_' . $index);
+
+        $this->getEventManager()->trigger(
+            new ClassPropertyCreatedEvent(
+                $class,
+                [
+                'label' => $newProperty->getLabel(),
+                'propertyUri' => $newProperty->getUri()
```

Comment:
Small indentation issue:

```suggestion
                    'label' => $newProperty->getLabel(),
                    'propertyUri' => $newProperty->getUri()
```